packages/adapter-next/src/sliceTemplates/CallToAction/index.ts

```diff
@@ -0,0 +1,271 @@
+import type { SharedSlice } from "@prismicio/types-internal/lib/customtypes";
+import { SharedSliceContent } from "@prismicio/types-internal/lib/content";
+
+export const mocks: SharedSliceContent[] = [
```

Comment:
IMO mocks should be optionally provided by the hook, library authors can edit fancy mocks if they want to, but it should be optional and default to auto mocking(?)

Reply:
I don't know, if screenshots are mandatory, I would like to end up with the same slice in the simulator. I think it will be easy enough to make them optional in next iteration

Reply:
TODO: Discuss that for a change in next iteration